### PR TITLE
New sport types

### LINKED ISSUE #3: supporting other activities
I would like to know what entails supporting other activities (most of which done have gps data (swim, pilates), and some do (mountain bike)). 

I created a small script to see how many of my activites would be converted:

```
#!/bin/bash
MYTMP=$(mktemp)
# echo "Temp file: $MYTMP"

for FILEJSON in ../Sport-sessions/*.json; do
    type=`cat $FILEJSON | jq -c '.sport_type_id'`
    echo $type >> $MYTMP
    # echo $type 
done

echo "Count and activity ids:"
sort $MYTMP | uniq -c | sort -bgr
```

outputs something like:
> Count and activity ids:
>     537 "18"
>     319 "34"
>     262 "31"
>     198 "23"
>     106 "4"
>      75 "1"
>      59 "27"
>      47 "3"
>      34 "5"
>      26 "7"
>      20 "19"
>       5 "15"

so I would like to get most of them covered. I am not proficient in js.
thanks!

### PR DESCRIPTION
**Pull request by mipapo**
Fixes #3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -214,7 +264,7 @@ const start = async ([exportPath, outputPath = `${process.cwd()}/export`]) => {
         .filter(
           session =>
             !(
-              ((type === 'ride' || type === 'run') &&
+              ((type === 'cycling' || type === 'run') &&
```
Comment (mipapo) [inline index.js]: Not sure if I need to adjust here anything else?

Comment (glennreyes) [reply index.js]: Can we keep name it "ride"? I think most folks are moving to Strava and I use this to name the activitity title here: https://github.com/glennreyes/runtastic-gpx/blob/master/index.js#L166-L170.  I know this part can be improved a lot, eg. ideally all cycling related activities are going to be titled into "Ride".

Comment (mipapo) [reply index.js]: Renamed it

Comment (glennreyes) [reply index.js]: Alternatively we can keep it cycling, but improve the naming I mentioned here: https://github.com/glennreyes/runtastic-gpx/blob/master/index.js#L166-L170

Comment (glennreyes) [reply index.js]: Opened https://github.com/glennreyes/runtastic-gpx/issues/6 for this 👍

**Pull request merged**